### PR TITLE
Fix attributes of Greater Travel Boots modifier

### PR DESCRIPTION
**Pull request by Trildar**

### CODE REVIEW

File: game/scripts/vscripts/items/farming/greater_travel_boots.lua [hunk 1]
```diff
@@ -139,6 +127,18 @@ function modifier_item_greater_travel_boots:IsHidden()
...
+function modifier_item_greater_travel_boots:IsDebuff()
+  return false
+end
+
+function modifier_item_greater_travel_boots:IsPurgable()
+  return false
+end
+
+function modifier_item_greater_travel_boots:GetAttributes()
+  return MODIFIER_ATTRIBUTE_MULTIPE
```
Comment (Haganeko) [inline game/scripts/vscripts/items/farming/greater_travel_boots.lua]: MULTIPE instead of MULTIPLE.

**Comment by chrisinajar** [comment]
Why should travel boots stack, and why aren't they hidden anymore?

File: game/scripts/vscripts/items/farming/greater_travel_boots.lua (hunk 1 above)
Comment (chrisinajar) [inline game/scripts/vscripts/items/farming/greater_travel_boots.lua]: `    game/scripts/vscripts/items/farming/greater_travel_boots.lua:139:10: (W113) accessing undefined variable MODIFIER_ATTRIBUTE_MULTIPE`
<3 travis

**Comment by Trildar** [comment]
Doesn't actually need to stack I suppose, but non-stacking boots movespeed is enforced by the modifier property already anyway, and passive item modifiers ought to stack in general. Probably less chance for it interacting problematically with the intrinsic multiplexer that way too.

**Pull request merged**